src/app/components/album/album.ui

```diff
@@ -51,6 +51,18 @@
         </style>
       </object>
     </child>
+    <child>
+      <object class="GtkLabel" id="year_label">
+        <property name="label">Year</property>
+        <property name="justify">center</property>
+        <property name="wrap">1</property>
+        <property name="wrap-mode">word-char</property>
+        <property name="max-width-chars">1</property>
```

Comment:
```suggestion
        <property name="max-width-chars">1</property>
        <property name="sensitive">0</property>
```

This will grey the year out, helping with the UI hierarchy.

Reply:
Also, github doesn't let me comment there, but the `spacing` of AlbumWidget seems a bit too spacious, 6 looks a bit better to me

Reply:
Thanks for reviewing! It works:

![image](https://user-images.githubusercontent.com/5410901/145037466-cecf043c-765b-4cb8-b8e0-21f9781462eb.png)

With spacing = 2, just to make sure things are working (my eyeballs couldn't confidently tell two pixels):
![image](https://user-images.githubusercontent.com/5410901/145038115-9d6e9612-67b4-4610-a670-11a4515350a1.png)

What about the details view?
![image](https://user-images.githubusercontent.com/5410901/145037937-7f762283-f8d7-4d1e-936b-a19cfd702c21.png)

Reply:
Details gonna recieve a facelift soon enough anyway, so I think just making it insensitive for now is fine.